packages/commonwealth/client/scripts/views/components/component_kit/new_designs/CWTagInput/CWTagInput.scss

```diff
@@ -0,0 +1,96 @@
+@import '../../../../../../styles/shared.scss';
+
+
+.CWTagInputContainer {
+  position: relative;
+  width: 100%;
+
+  .CWTagInput {
+    display: flex;
+    align-items: center;
+    // variable styles (TODO: use css variables)
```

Comment:
what this relates to?

Reply:
I forgot to update the CSS to use the SCSS vars we have